com.unity.shadergraph/Editor/Resources/StickyNote.uss

```diff
@@ -10,7 +10,6 @@
     position:absolute;
     flex-direction:column;
     align-items:stretch;
-    --layer:-100;
```

Comment:
This caused a really nasty selection bug. When pasting a sticky note it would always lose focus on the pasted node even when manually added to the selection. This was because the node would be newly created and then the layout would change the layer which would update it in the hierarchy and remove it from the selection (deep inside a layout callstack under Repaint). After talking with @Nightmask3 this seemed valid to move into the sticky note constructor which also fixed the problem.